src/Domain/Item/ItemService.ts

```diff
@@ -1,22 +1,28 @@
 import * as dayjs from 'dayjs'
 import { inject, injectable } from 'inversify'
 import TYPES from '../../Bootstrap/Types'
+import { TimerInterface } from '../Time/TimerInterface'
 import { GetItemsDTO } from './GetItemsDTO'
 
 import { GetItemsResult } from './GetItemsResult'
 import { Item } from './Item'
+import { ItemConflict } from './ItemConflict'
+import { ItemHash } from './ItemHash'
 import { ItemQuery } from './ItemQuery'
 import { ItemRepositoryInterface } from './ItemRepositoryInterface'
 import { ItemServiceInterface } from './ItemServiceInterface'
+import { SaveItemsDTO } from './SaveItemsDTO'
 import { SaveItemsResult } from './SaveItemsResult'
 
 @injectable()
 export class ItemService implements ItemServiceInterface {
   private readonly DEFAULT_ITEMS_LIMIT = 100000
   private readonly SYNC_TOKEN_VERSION = 2
+  private readonly MIN_CONFLICT_INTERVAL_MICROSECONDS = 1000000
```

Comment:
This should actually be 1,000 microseconds ([Ruby](https://github.com/standardnotes/syncing-server/pull/183/files#diff-93ff14120c01a9761658c9a26922385b4982682e4b7c29776a48443ef8320927R35)). What we are saying here is "if two items updated_at differ by greater than 1,000 microseconds, create a conflict."

Reply:
I was aiming to apply "fix more precise sync date conflict handling" in a separate task/PR - so that at first we have what's in current Ruby's develop so nothing gets mixed up by accident

Reply:
Will fix 👍